pkgs/development/python-modules/wn/default.nix

```diff
@@ -0,0 +1,39 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pytest
+, flit-core
+, requests
+, tomli
+}:
+
+buildPythonPackage rec {
+  pname = "wn";
+  version = "0.9.2";
+  format = "pyproject";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-TghCKPKLxRTpvojmZi8tPGmU/D2W+weZl64PArAwDCE=";
+  };
+
+  nativeBuildInputs = [ flit-core ];
+
+  propagatedBuildInputs = [ requests tomli ];
```

Comment:
It is nicer to have one dependency per line, since it makes it easier to use `git blame`.